src/vivarium_ihme_obesity_washout/components/observer.py

```diff
@@ -1,104 +1,79 @@
+        self.program_start = pd.Timestamp(**self.washout_config.program_start.to_dict())
+        if self.start_time > self.program_start:
             raise ValueError(f'The washout program must begin after the start of the simulation. '
                              f'Your simulation begins: {self.start_time} while the '
-                             f'washout program begins {program_start}.')
+                             f'washout program begins {self.program_start}.')
 
-        self.washout_period_end = program_start + pd.Timedelta(days=self.config.washout.duration*365.25)
-        self.initial_pop_entrance_time = self.start_time - self.step_size()
-        self.age_bins = get_age_bins(builder)
+        self.bmi_exposure = builder.value.get_value('high_body_mass_index_in_adults.exposure')
 
-        diseases = builder.components.get_components_by_type((DiseaseModel, RiskAttributableDisease))
-        self.causes = [c.state_column for c in diseases] + ['other_causes']
+    def split_bmi(self, index: pd.Index) -> Dict[str, pd.Index]:
+        """Splits an index into groups based on BMI exposure.
 
-        life_expectancy_data = builder.data.load("population.theoretical_minimum_risk_life_expectancy")
-        self.life_expectancy = builder.lookup.build_table(life_expectancy_data, key_columns=[],
-                                                          parameter_columns=[('age', 'age_group_start',
-                                                                              'age_group_end')])
+        Parameters
+        ----------
+        index
+            The population index to split.
 
-        self.bmi_exposure = builder.value.get_value('high_body_mass_index_in_adults.exposure')
+        Returns
+        -------
+            A mapping between bmi group names and the index of the population
+            in those bmi groups.
 
-        columns_required = ['tracked', 'alive', 'entrance_time', 'exit_time', 'cause_of_death',
-                            'years_of_life_lost', 'age']
-        if self.config.by_sex:
-            columns_required += ['sex']
-        self.population_view = builder.population.get_view(columns_required)
+        """
+        bmi = self.bmi_exposure(index)
+        bins = self.washout_config.bmi_bins
+        groups = {f'bmi_group_{lower}_to_{upper}': bmi.loc[(lower <= bmi) & (bmi < upper)].index
+                  for lower, upper in zip(bins[:-1], bins[1:])}
 
-        builder.value.register_value_modifier('metrics', self.metrics)
+        return groups
 
-    def get_bmi_categories(self):
-        bins = self.config.categories
-        lower_bounds = bins.copy()[:-1]
-        lower_bounds.insert(0, 0)
-        names = [f'bmi_group_{c[0]}_to_{c[1]}' for c in zip(lower_bounds, bins)]
-        bins.insert(0, 0)
-        return names, bins
+    def eligible(self, pop: pd.DataFrame) -> pd.Index:
+        """Gets the observation eligible index."""
+        duration = pd.Timedelta(days=self.washout_config.duration)
```

Comment:
I think you need to multiply self.washout_config.duration by 365.25 here - it looks like it's specified in years in the config